### DISCUSSION TITLE
How do I configure a containerd mirror?

### DISCUSSION THREAD

**Discussion by jdolitsky** [lima-vm/lima#666] [Q&A]
I'm attempting to use `mirror.gcr.io`, since I'm getting issues downloading images form `docker.io`

What is the recommended way to go about this with lima?

I tried doing something like this in an ubuntu VM, but it doesn't appear to work properly:

```
provision:
- mode: system
  script: |
    #!/bin/bash
    set -eux -o pipefail
    cat << 'EOF' >> /etc/containerd/config.toml
      [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
        [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
          endpoint = ["https://mirror.gcr.io"]
    EOF
    systemctl restart containerd
```

**Comment by afbjorklund** [comment]
I think you want to configure the host.<mirror>:
https://github.com/containerd/containerd/blob/main/docs/hosts.md#host-fields-in-the-toml-table-format

The configuration you posted is for Kubernetes CRI.
